site/docs/master/aws-config.md

```diff
@@ -292,13 +292,10 @@ velero install \
     --pod-annotations iam.amazonaws.com/role=arn:aws:iam::<AWS_ACCOUNT_ID>:role/<VELERO_ROLE_NAME> \
     --provider aws \
     --bucket $BUCKET \
-    --secret-file ./credentials-velero \
     --backup-location-config region=$REGION \
     --snapshot-location-config region=$REGION
 ```
```

Comment:
we'd need to specify `--no-secret` here, right?

Reply:
Yep, missed that.